server/src/com/google/refine/Refine.java

```diff
@@ -494,12 +494,42 @@ public void actionPerformed(ActionEvent e) {
     
     private void openBrowser() {
         if (!Desktop.isDesktopSupported()) {
```

Comment:
This returns true in every operating system apparently, because the class itself is supported. We need to specifically determine if the action we want to perform is.
According to this [link](https://stackoverflow.com/questions/5226212/how-to-open-the-default-webbrowser-using-java/21676290#21676290), GNOME libraries are required for the `Desktop.browse` action. Unfortunately, it is possible to use Linux without them (for example, I use KDE as my desktop env and this solution does not work for me).

A more precise approach will be to replace this check with this:
```java
if(!Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
```
The `openBrowser()` method is indeed very necessary.